ui/src/components/table/TableWrapper.tsx

```diff
@@ -174,10 +213,12 @@ function TableWrapper({
                 if (
                     tableFields.includes(vv) &&
                     typeof data[v][vv] === 'string' &&
-                    data[v][vv].toLowerCase().includes(searchText.toLowerCase().trim()) &&
+                    (data[v][vv] as string)
+                        .toLowerCase()
+                        .includes(searchText.toLowerCase().trim()) &&
                     !found
                 ) {
-                    arr.push(data[v]);
+                    arr.push(data[v] as Record<string, string>);
```

Comment:
something fishy is here. We declare Record<string, string>[] as a result of findings. But instead we put there Record<string, AcceptableFormValueOrNull>

Reply:
My point is that `as` as smelly as `any` - I would avoid it as all cost because it disables TS

Reply:
true